instrumentation/spring/spring-scheduling-3.1/javaagent/src/test/java/SpringSchedulingTest.java

```diff
@@ -0,0 +1,197 @@
+/*
```

Comment:
usually we place java tests in the same package as the instrumentation code, which here would be `io.opentelemetry.javaagent.instrumentation.spring.scheduling.v3_1`

Reply:
Moved tests to `io.opentelemetry.javaagent.instrumentation.spring.scheduling.v3_1` package